generator/simplecpp/simplecpp.cpp

```diff
@@ -3116,13 +3116,40 @@ static std::string getIncludePathFileName(const std::string &includePath, const
     return path + header;
 }
 
+#ifdef __APPLE__
+static std::string get_apple_framework_relative_path(const std::string& header)
```

Comment:
Please try to use the same naming style (e.g. camel case) as the rest of the code.